test-requirements.txt

```diff
@@ -7,6 +7,5 @@ flake8<4.1
 flake8-copyright<0.3
 juju<2.10
 pytest<6.3
-pytest-operator<0.17.0
```

Comment:
I think you meant to delete the pin, not the entire dependency

Reply:
As discussed, the dependency is also present [in tox](https://github.com/canonical/dex-auth-operator/blob/main/tox.ini#L29)